changelogs/fragments/682_update_ci_20250929.yml

```diff
@@ -0,0 +1,3 @@
+trivial:
+  - Updatng AZP CI matrix to ignore ansible-bad-import-from on six(https://github.com/ansible-collections/ansible.posix/pull/682).
+  - Removing sandalone ansible-lint test it already covered by ansible-test.
```

Comment:
ansible-test does not run ansible-lint.

A very stripped down version of ansible-lint runs during the Galaxy importer test, but even if that fails the importer tests succeeds. So basically ansible-lint is no longer run with this PR. (Assuming there is no ansible-lint run hidden somewhere else.)

If this is intended, it's OK from my side to merge this PR.

Reply:
Thank you for pointing that out! I seem to have completely misunderstood about ansible-test behavior.

1. I have reverted the Lint test changes and will perform ansible-lint test as before.
2. I noticed that the errors regarding ignoring `ansible-bad-import-from` in `ignore-2.20.txt` could be excluded from ansible-lint configuration. Therefore, excluded the `tests/sanity/` directory from ansible-lint testing. 

I think it's become more rational than before. Please share your opinion.